app/models/work.rb

```diff
@@ -401,7 +401,18 @@ def post_curation_s3_resources
 
     s3_resources
   end
-  alias post_curation_uploads post_curation_s3_resources
+
+  # Returns the files in post-curation for the work
+  def post_curation_uploads(force_post_curation: false)
```

Comment:
```suggestion
  def post_curation_uploads(force_post_curation: true)
```

Reply:
I didn't want to change the existing behavior on this PR since I am not sure what depends on the current behavior.